app-doc/gimp-help/Manifest

```diff
@@ -1,2 +1 @@
 DIST gimp-help-2.10.0.tar.bz2 187092180 BLAKE2B 29a7fbe3fcb6f3210aeda4bc514d7e0eeca05f9fabd3b9022c7ed1e5a9432876cc3bf530632648e2eb50509add4e387106d9705a1c17469e33496daec737d21b SHA512 2f6d5e7cf0c3b4960c92074ef02d51a1d089c148f6453a66e453a3084cdede280f72c6b77d35f66777adbda8aa273ec2a472fb538142e51bb263c3077eec473a
-DIST gimp-help-2.8.2.tar.bz2 158504580 BLAKE2B d0023a4290879f97e3365f1208427997d1facc09a7ed4ac0a274615e9eff378c9a4da0d7c5190bc24ad27c2b03d6c513003297d835562b0fa3e02685443d15e9 SHA512 3afdaa8b2f005d939d039d2e22c8306145e7b7acaca5ef89279998e87931e99d1fe607f1337f7796d667d5c6be456ba976b3c8a937c9dd6b1888d7158cfbed67
```

Comment:
This can't be dropped because newer one is not stabilized, merged the python3 one.